### PR TITLE
benchmarks/aggregate: add tests + use oldest benchmarked Inductor as baseline

### PR DESCRIPTION
**Pull request by cota**
See commit logs and updated readme for rationale.

### CODE REVIEW

**Comment by frgossen** [comment]
This is great. Thank you!

File: benchmarks/test/Makefile [hunk 1]
```diff
@@ -0,0 +1,21 @@
+TEST_ARGS = $(shell echo $@ | perl -pe 's/([^.]*)\.([^.]*)\.([^.]*)\.test/--accelerator=$$1 --test=$$2 --report=$$3/')
```
Comment (frgossen) [inline benchmarks/test/Makefile:1]: @JackCaoG , is this the way to add such a test or is there a more idiomatic way to add such tests in PyTorch/XLA?

Comment (JackCaoG) [reply benchmarks/test/Makefile:1]: we run out CI in https://github.com/pytorch/xla/blob/1ea23741cadbe754728d2dac42fcfcb2718806dc/.circleci/common.sh#L128

that script will run python tests.

Comment (cota) [reply benchmarks/test/Makefile:1]: Thanks! Sent https://github.com/pytorch/xla/pull/6035 as a follow-up PR

**Review by frgossen — APPROVED** [review]

**Pull request merged**